controllers/component_build_controller.go

```diff
@@ -217,6 +217,9 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, nil
 	}
 
+	log = log.WithValues("ComponentGitSource", component.Spec.Source.GitSource)
+	ctx = ctrllog.IntoContext(ctx, log)
+
```

Comment:
Is this necessary for the update?

Reply:
Not really. It's just a minor improvement for logging.

Reply:
I would also include it in separate commit in that case

Reply:
ok